src/displayapp/screens/Dice.h

```diff
@@ -29,8 +28,9 @@ namespace Pinetime::Applications::Screens {
     Pinetime::System::SystemTask& systemTask;
     Controllers::MotionController& motionController;
     Controllers::MotorController& motorController;
+    Controllers::DateTime& dateTimeController;
```

Comment:
There's no need to store a reference to `dateTimeController` inside of this object. It's only used once inside of the constructor.

On that note `Dice` doesn't need to know anything about date/time at all. It simply needs a seed value which in this case coincidentally gets supplied with the current time. I'd personally suggest changing the constructor parameter to be an `unsigned seed` instead and passing in `dateTimeController.CurrentDateTime().time_since_epoch().count()` to it.

Reply:
@jonvmey You're right -- this is a much better way to do it.  Implemented in https://github.com/JF002/InfiniTime/pull/565/commits/75c7e280842f08b91be91de121d600ec5f87defd